internal/ingress/controller/controller.go

```diff
@@ -124,10 +124,8 @@ func (n *NGINXController) syncIngress(interface{}) error {
 	ings := n.store.ListIngresses(nil)
 	hosts, servers, pcfg := n.getConfiguration(ings)
 
-	if n.isLeader() {
-		klog.V(2).Infof("Updating ssl expiration metrics.")
-		n.metricCollector.SetSSLExpireTime(servers)
-	}
+	klog.V(2).Infof("Updating ssl expiration metrics.")
+	n.metricCollector.SetSSLExpireTime(servers)
```

Comment:
Why did you change this?

Reply:
Because the validation is in the metric collector (this PR)

Reply:
Can you link where it is? I'd have expected to see it in https://github.com/kubernetes/ingress-nginx/pull/4251/files#diff-71033b20e83348da90cc4ad44522573dR156.

Reply:
fixed